bin/run_tertiary_workflow.sh

```diff
@@ -72,6 +72,11 @@ if [ -n "$cell_type_field" ]; then
     sed -i "s/CELL_TYPE_FIELD/$cell_type_field/" $parameters_yaml 
 fi
 
+# If the batch variable is set, then tell the workflow about it. Otherwise just
+# unset it so any batch-adjustment steps just 'pass through'.
+
+sed -i "s/BATCH_VARIABLE/$batch_field/" $parameters_yaml 
```

Comment:
Sorry being anal here, can we use the same variable name for a replacement like `BATCH_FIELD` like how you have declared above for `CELL_TYPE_FIELD`